<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Hao He" />


<title>Variant_plot</title>

<script src="site_libs/header-attrs-2.6/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">QTL_analysis_for_Crichton</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/KumarLabJax/QTL_analysis_for_Crichton">
    <span class="fas fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Variant_plot</h1>
<h4 class="author">Hao He</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-03-15
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>QTL_analysis_for_Crichton/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20221224code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20221224)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20221224code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20221224)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsghpgZ6sSPKNx11v77sSgITOIvpJI94FN62ln5MXgithubcomKumarLabJaxQTLanalysisforCrichtontree9e3e4c4c3b26d4a63fa9b2e4c787058e9cb19a9btargetblank9e3e4c4a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://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@github.com/KumarLabJax/QTL_analysis_for_Crichton/tree/9e3e4c4c3b26d4a63fa9b2e4c787058e9cb19a9b" target="_blank">9e3e4c4</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsghpgZ6sSPKNx11v77sSgITOIvpJI94FN62ln5MXgithubcomKumarLabJaxQTLanalysisforCrichtontree9e3e4c4c3b26d4a63fa9b2e4c787058e9cb19a9btargetblank9e3e4c4a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://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@github.com/KumarLabJax/QTL_analysis_for_Crichton/tree/9e3e4c4c3b26d4a63fa9b2e4c787058e9cb19a9b" target="_blank">9e3e4c4</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/plot.R
    Untracked:  data/GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snpEff.vcf
    Untracked:  data/GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snpEff.vcf.idx
    Untracked:  ex/
    Untracked:  output/GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.pdf
    Untracked:  output/GES15-07029-D-WT144G8N2F703F.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.pdf
    Untracked:  output/WT144_effect_for_marker__int_chr5-136332619-.-C-A.pdf
    Untracked:  output/WT144_effect_for_marker_plots_TotalDist_vs_TestAge_by_marker.pdf
    Untracked:  output/WT144_plot-Intercept.pdf
    Untracked:  output/WT144_plot-Slope.pdf
    Untracked:  output/WT144_plot-TestAge.x.pdf
    Untracked:  output/WT144_plot-TestAge.y.pdf
    Untracked:  output/WT144_plot-TotalDist_1.pdf
    Untracked:  output/WT144_plot-TotalDist_2.pdf
    Untracked:  output/WT144_plot-TotalDist_3.pdf
    Untracked:  output/WT144_plot-base.x.pdf
    Untracked:  output/WT144_plot-base.y.pdf
    Untracked:  output/WT144_variants_nointersect_noindels_homo.pdf
    Untracked:  output/figure_panel.pdf
    Untracked:  output/qtl.out.obj.mixedmodel.RData
    Untracked:  output/qtl.out.obj.mixedmodel.final.RData
    Untracked:  output/wt144_chr14_qtlinterval.pdf
    Untracked:  output/wt144_chr5_qtlinterval.pdf

Unstaged changes:
    Modified:   analysis/workflow_proc.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Variant_plot.Rmd</code>) and HTML (<code>docs/Variant_plot.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://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@github.com/KumarLabJax/QTL_analysis_for_Crichton/blob/9e3e4c4c3b26d4a63fa9b2e4c787058e9cb19a9b/analysis/Variant_plot.Rmd" target="_blank">9e3e4c4</a>
</td>
<td>
xhyuo
</td>
<td>
2023-03-15
</td>
<td>
Variant_plot
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="library" class="section level3">
<h3>Library</h3>
<pre class="r"><code>library(tidyverse)
library(parallel)
library(Rsamtools)
library(data.table)
library(ensemblVEP)
library(karyoploteR)
library(vcfR)
library(biomaRt)
library(regioneR)
library(qtl2)</code></pre>
</div>
<div id="homo-variants-plot-for-two-wt144-wgs" class="section level3">
<h3>Homo variants plot for Two WT144 WGS</h3>
<pre class="r"><code># GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snp --------
#read vcf GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.recode.vcf
vcf.file1 = &quot;/projects/compsci/legacy/USERS/peera/Kumar_lab/WT144_markers/GES15-07028-C-WT144G8N2F70426M.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.recode.vcf&quot;
vcf1 = read.vcfR(vcf.file1)</code></pre>
<pre><code>Scanning file to determine attributes.
File attributes:
  meta lines: 105
  header_line: 106
  variant count: 1955
  column count: 10

Meta line 105 read in.
All meta lines processed.
gt matrix initialized.
Character matrix gt created.
  Character matrix gt rows: 1955
  Character matrix gt cols: 10
  skip: 0
  nrows: 1955
  row_num: 0

Processed variant 1000
Processed variant: 1955
All variants processed</code></pre>
<pre class="r"><code>#snp
snp1 &lt;- vcf1@fix %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(1:5) %&gt;%
  dplyr::mutate(ID = case_when(
    is.na(ID) ~ paste0(CHROM, &quot;_&quot;, POS),
    TRUE ~ ID
  )) %&gt;%
  dplyr::mutate(pos0 = as.numeric(POS)-1) %&gt;%
  dplyr::select(CHROM, pos0, POS) %&gt;%
  toGRanges()

# GES15-07029-D-WT144G8N2F703F.final_variants_filtered_dbsnp --------
#read vcf GES15-07029-D-WT144G8N2F703F.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.recode.vcf
vcf.file2 = &quot;/projects/compsci/legacy/USERS/peera/Kumar_lab/WT144_markers/GES15-07029-D-WT144G8N2F703F.final_variants_filtered_dbsnp_snpEff_nointersect_noindels_homo.recode.vcf&quot;
vcf2 = read.vcfR(vcf.file2)</code></pre>
<pre><code>Scanning file to determine attributes.
File attributes:
  meta lines: 105
  header_line: 106
  variant count: 1787
  column count: 10

Meta line 105 read in.
All meta lines processed.
gt matrix initialized.
Character matrix gt created.
  Character matrix gt rows: 1787
  Character matrix gt cols: 10
  skip: 0
  nrows: 1787
  row_num: 0

Processed variant 1000
Processed variant: 1787
All variants processed</code></pre>
<pre class="r"><code>#snp
snp2 &lt;- vcf2@fix %&gt;%
  as.data.frame() %&gt;%
  dplyr::select(1:5) %&gt;%
  dplyr::mutate(ID = case_when(
    is.na(ID) ~ paste0(CHROM, &quot;_&quot;, POS),
    TRUE ~ ID
  )) %&gt;%
  dplyr::mutate(pos0 = as.numeric(POS)-1) %&gt;%
  dplyr::select(CHROM, pos0, POS) %&gt;%
  toGRanges()

#plot
pp &lt;- getDefaultPlotParams(plot.type=2)
pp$data1height &lt;- 140
pp$data2height &lt;- 140
pp$topmargin &lt;- 300
kp &lt;- plotKaryotype(genome = &quot;mm10&quot;, plot.type = 2, plot.params = pp)
kpAddMainTitle(kp, main=&quot;WT144 homozygous variants&quot;, cex = 1.2)
kpDataBackground(kp, data.panel = 1, color = &quot;white&quot;)
kpDataBackground(kp, data.panel = 2, color = &quot;white&quot;)
kpPlotRegions(kp, data=snp1, col=&quot;blue&quot;, avoid.overlapping = FALSE, r0 = 0, r1 = 0.8, data.panel = 1)
kpPlotRegions(kp, data=snp2, col=&quot;red&quot;,  avoid.overlapping = FALSE, r0 = 0, r1 = 0.8, data.panel = 2)</code></pre>
<p><img src="figure/Variant_plot.Rmd/Homo%20variants%20plot%20for%20Two%20WT144%20WGS-1.png" width="1056" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#save plot
pdf(file = &quot;output/WT144_variants_nointersect_noindels_homo.pdf&quot;, width = 11, height = 9)
pp &lt;- getDefaultPlotParams(plot.type=2)
pp$data1height &lt;- 140
pp$data2height &lt;- 140
pp$topmargin &lt;- 300
kp &lt;- plotKaryotype(genome = &quot;mm10&quot;, plot.type = 2, plot.params = pp)
kpAddMainTitle(kp, main=&quot;WT144 homozygous variants&quot;, cex = 1.2)
kpDataBackground(kp, data.panel = 1, color = &quot;white&quot;)
kpDataBackground(kp, data.panel = 2, color = &quot;white&quot;)
kpPlotRegions(kp, data=snp1, col=&quot;blue&quot;, avoid.overlapping = FALSE, r0 = 0, r1 = 0.8, data.panel = 1)
kpPlotRegions(kp, data=snp2, col=&quot;red&quot;,  avoid.overlapping = FALSE, r0 = 0, r1 = 0.8, data.panel = 2)
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
</div>
<div id="wt144-qtl-interval" class="section level3">
<h3>WT144 QTL interval</h3>
<pre class="r"><code>#genes in the qtl region
query_variants &lt;- create_variant_query_func(&quot;/projects/compsci/vmp/USERS/heh/DO_Opioid/data/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/projects/compsci/vmp/USERS/heh/DO_Opioid/data/mouse_genes_mgi.sqlite&quot;)

#chr5 interval-----------------------------------------------------------------------
#the interval for chr5 is 72.40547 to 77.76853 cM, in bp is 133052642 to 138956439
#Ssc4d at chr5 135.9602 - 135.9745; MGI:MGI:1924709
chr5_gene &lt;- query_genes(chr = 5, 129, 139) %&gt;%
  dplyr::filter(!str_detect(Name, &quot;^Gm&quot;)) # remove gene starting with Gm
  # dplyr::mutate(Name = case_when(
  #   str_detect(Name, &quot;^Gm&quot;) ~ &quot;&quot;,
  #   TRUE ~ as.character(Name)
  #   ))

#variant
vcf.file3 = &quot;/projects/compsci/legacy/USERS/peera/Kumar_lab/WT144_markers/final_list_of_markers.vcf&quot;
vcf3 = read.vcfR(vcf.file3)</code></pre>
<pre><code>Scanning file to determine attributes.
File attributes:
  meta lines: 108
  header_line: 109
  variant count: 340
  column count: 11

Meta line 108 read in.
All meta lines processed.
gt matrix initialized.
Character matrix gt created.
  Character matrix gt rows: 340
  Character matrix gt cols: 11
  skip: 0
  nrows: 340
  row_num: 0

Processed variant: 340
All variants processed</code></pre>
<pre class="r"><code>#snp
chr5.region &lt;- vcf3@fix %&gt;%
  as.data.frame() %&gt;%
  dplyr::mutate(POS = as.numeric(POS)) %&gt;%
  dplyr::mutate(eff = str_match(INFO, &quot;;EFF=\\s*(.*?)\\s*;&quot;)[,2]) %&gt;%
  dplyr::mutate(anno = gsub(&quot;\\(.*&quot;, &quot;&quot;, eff)) %&gt;%
  dplyr::filter(CHROM == &quot;chr5&quot;) %&gt;%
  dplyr::filter(between(POS, 129*1e6, 139*1e6)) %&gt;%
  dplyr::mutate(anno = as.factor(anno))


# 2 x 1 panels; adjust margins
old_mfrow &lt;- par(&quot;mfrow&quot;)
old_mar &lt;- par(&quot;mar&quot;)
on.exit(par(mfrow=old_mfrow, mar=old_mar))
layout(rbind(1,2), heights=c(2, 4))
top_mar &lt;- bottom_mar &lt;- old_mar
top_mar &lt;- c(0.01, 10, 2, 2)
bottom_mar &lt;- c(5.10, 10, 0.01, 2)

par(mar=top_mar)
#Create the base plot
plot(chr5.region$POS, chr5.region$anno, type = &quot;n&quot;, xaxt = &quot;n&quot;,
     xlab = &quot;POS&quot;,ylab = &quot;&quot;, yaxt = &quot;n&quot;, main = &quot;WT144 Chr5 QTL interval&quot;)
# Add points to the plot
points(chr5.region$POS, chr5.region$anno, pch = &quot;|&quot;, cex = 1, col = &quot;red&quot;)
box()  # Add a box around the plot
# Change the y-axis tick labels
axis(2, 1:length(levels(chr5.region$anno)), labels = levels(chr5.region$anno), las = 1, cex.axis = 0.65)
#bottom
par(mar=bottom_mar)
plot_genes(chr5_gene, bgcolor=&quot;white&quot;)</code></pre>
<p><img src="figure/Variant_plot.Rmd/WT144%20QTL%20interval-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code># save the plot
pdf(file = &quot;output/wt144_chr5_qtlinterval.pdf&quot;, width = 10, height = 8)
# 2 x 1 panels; adjust margins
old_mfrow &lt;- par(&quot;mfrow&quot;)
old_mar &lt;- par(&quot;mar&quot;)
on.exit(par(mfrow=old_mfrow, mar=old_mar))
layout(rbind(1,2), heights=c(2, 4))
top_mar &lt;- bottom_mar &lt;- old_mar
top_mar &lt;- c(0.01, 10, 2, 2)
bottom_mar &lt;- c(5.10, 10, 0.01, 2)

par(mar=top_mar)
#Create the base plot
plot(chr5.region$POS, chr5.region$anno, type = &quot;n&quot;, xaxt = &quot;n&quot;,
     xlab = &quot;POS&quot;,ylab = &quot;&quot;, yaxt = &quot;n&quot;, main = &quot;WT144 Chr5 QTL interval&quot;)
# Add points to the plot
points(chr5.region$POS, chr5.region$anno, pch = &quot;|&quot;, cex = 1, col = &quot;red&quot;)
box()  # Add a box around the plot
# Change the y-axis tick labels
axis(2, 1:length(levels(chr5.region$anno)), labels = levels(chr5.region$anno), las = 1, cex.axis = 0.65)
#bottom
par(mar=bottom_mar)
plot_genes(chr5_gene, bgcolor=&quot;white&quot;)
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
<pre class="r"><code>#chr14 interval-----------------------------------------------------------------------
#the interval for chr14 is 26.14972 to 36.07219 cM, in bp is 50642778 to 69691477
#Kpna3 at chr14 61.36519 - 61.43995; MGI:MGI:1100863
chr14_gene &lt;- query_genes(chr = 14, 59.642778, 69.691477) %&gt;%
  dplyr::filter(!str_detect(Name, &quot;^Gm&quot;)) # remove gene starting with Gm
#snp
chr14.region &lt;- vcf3@fix %&gt;%
  as.data.frame() %&gt;%
  dplyr::mutate(POS = as.numeric(POS)) %&gt;%
  dplyr::mutate(eff = str_match(INFO, &quot;;EFF=\\s*(.*?)\\s*;&quot;)[,2]) %&gt;%
  dplyr::mutate(anno = gsub(&quot;\\(.*&quot;, &quot;&quot;, eff)) %&gt;%
  dplyr::filter(CHROM == &quot;chr14&quot;) %&gt;%
  dplyr::filter(between(POS, 59.642778*1e6, 69.691477*1e6)) %&gt;%
  dplyr::mutate(anno = as.factor(anno))

# 2 x 1 panels; adjust margins
old_mfrow &lt;- par(&quot;mfrow&quot;)
old_mar &lt;- par(&quot;mar&quot;)
on.exit(par(mfrow=old_mfrow, mar=old_mar))
layout(rbind(1,2), heights=c(0.5, 2))
top_mar &lt;- bottom_mar &lt;- old_mar
top_mar &lt;- c(0.01, 10, 2, 2)
bottom_mar &lt;- c(5.10, 10, 0.01, 2)

par(mar=top_mar)
#Create the base plot
plot(chr14.region$POS, chr14.region$anno, type = &quot;n&quot;, xaxt = &quot;n&quot;,
     xlab = &quot;POS&quot;,ylab = &quot;&quot;, yaxt = &quot;n&quot;, main = &quot;WT144 Chr14 QTL interval&quot;)
# Add points to the plot
points(chr14.region$POS, chr14.region$anno, pch = &quot;|&quot;, cex = 1, col = &quot;red&quot;)
box()  # Add a box around the plot
# Change the y-axis tick labels
axis(2, 1:length(levels(chr14.region$anno)), labels = levels(chr14.region$anno), las = 1, cex.axis = 0.65)
#bottom
par(mar=bottom_mar)
plot_genes(chr14_gene, bgcolor=&quot;white&quot;)</code></pre>
<p><img src="figure/Variant_plot.Rmd/WT144%20QTL%20interval-2.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code># save the plot
pdf(file = &quot;output/wt144_chr14_qtlinterval.pdf&quot;, width = 10, height = 8)
# 2 x 1 panels; adjust margins
old_mfrow &lt;- par(&quot;mfrow&quot;)
old_mar &lt;- par(&quot;mar&quot;)
on.exit(par(mfrow=old_mfrow, mar=old_mar))
layout(rbind(1,2), heights=c(0.5, 2))
top_mar &lt;- bottom_mar &lt;- old_mar
top_mar &lt;- c(0.01, 10, 2, 2)
bottom_mar &lt;- c(5.10, 10, 0.01, 2)

par(mar=top_mar)
#Create the base plot
plot(chr14.region$POS, chr14.region$anno, type = &quot;n&quot;, xaxt = &quot;n&quot;,
     xlab = &quot;POS&quot;,ylab = &quot;&quot;, yaxt = &quot;n&quot;, main = &quot;WT144 Chr14 QTL interval&quot;)
# Add points to the plot
points(chr14.region$POS, chr14.region$anno, pch = &quot;|&quot;, cex = 1, col = &quot;red&quot;)
box()  # Add a box around the plot
# Change the y-axis tick labels
axis(2, 1:length(levels(chr14.region$anno)), labels = levels(chr14.region$anno), las = 1, cex.axis = 0.65)
#bottom
par(mar=bottom_mar)
plot_genes(chr14_gene, bgcolor=&quot;white&quot;)
dev.off()</code></pre>
<pre><code>png 
  2 </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.2 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] qtl2_0.24                   biomaRt_2.46.3             
 [3] vcfR_1.12.0                 karyoploteR_1.16.0         
 [5] regioneR_1.22.0             ensemblVEP_1.32.1          
 [7] VariantAnnotation_1.36.0    SummarizedExperiment_1.20.0
 [9] Biobase_2.50.0              MatrixGenerics_1.2.1       
[11] matrixStats_0.58.0          data.table_1.13.6          
[13] Rsamtools_2.6.0             Biostrings_2.58.0          
[15] XVector_0.30.0              GenomicRanges_1.42.0       
[17] GenomeInfoDb_1.26.7         IRanges_2.24.1             
[19] S4Vectors_0.28.1            BiocGenerics_0.36.1        
[21] forcats_0.5.1               stringr_1.4.0              
[23] dplyr_1.0.4                 purrr_0.3.4                
[25] readr_1.4.0                 tidyr_1.1.2                
[27] tibble_3.0.6                ggplot2_3.3.3              
[29] tidyverse_1.3.0             workflowr_1.6.2            

loaded via a namespace (and not attached):
  [1] readxl_1.3.1             backports_1.2.1          Hmisc_4.4-2             
  [4] BiocFileCache_1.14.0     lazyeval_0.2.2           splines_4.0.3           
  [7] BiocParallel_1.24.1      digest_0.6.27            ensembldb_2.14.1        
 [10] htmltools_0.5.1.1        magrittr_2.0.1           checkmate_2.0.0         
 [13] memoise_2.0.0            BSgenome_1.58.0          cluster_2.1.1           
 [16] modelr_0.1.8             askpass_1.1              prettyunits_1.1.1       
 [19] jpeg_0.1-8.1             colorspace_2.0-0         blob_1.2.1              
 [22] rvest_0.3.6              rappdirs_0.3.3           haven_2.3.1             
 [25] xfun_0.21                crayon_1.4.1             RCurl_1.98-1.2          
 [28] jsonlite_1.7.2           ape_5.4-1                survival_3.2-7          
 [31] glue_1.4.2               gtable_0.3.0             zlibbioc_1.36.0         
 [34] DelayedArray_0.16.3      scales_1.1.1             bezier_1.1.2            
 [37] DBI_1.1.1                Rcpp_1.0.6               viridisLite_0.3.0       
 [40] progress_1.2.2           htmlTable_2.1.0          foreign_0.8-81          
 [43] bit_4.0.4                Formula_1.2-4            htmlwidgets_1.5.3       
 [46] httr_1.4.2               RColorBrewer_1.1-2       ellipsis_0.3.1          
 [49] pkgconfig_2.0.3          XML_3.99-0.5             nnet_7.3-15             
 [52] dbplyr_2.1.0             tidyselect_1.1.0         rlang_1.0.2             
 [55] later_1.1.0.1            AnnotationDbi_1.52.0     munsell_0.5.0           
 [58] cellranger_1.1.0         tools_4.0.3              cachem_1.0.4            
 [61] cli_2.3.0                generics_0.1.0           RSQLite_2.2.3           
 [64] broom_0.7.4              evaluate_0.14            fastmap_1.1.0           
 [67] yaml_2.2.1               knitr_1.31               bit64_4.0.5             
 [70] fs_1.5.0                 AnnotationFilter_1.14.0  nlme_3.1-152            
 [73] whisker_0.4              xml2_1.3.2               compiler_4.0.3          
 [76] rstudioapi_0.13          curl_4.3                 png_0.1-7               
 [79] reprex_1.0.0             stringi_1.5.3            highr_0.8               
 [82] GenomicFeatures_1.42.3   memuse_4.1-0             lattice_0.20-41         
 [85] ProtGenerics_1.22.0      Matrix_1.3-2             permute_0.9-5           
 [88] vegan_2.5-7              vctrs_0.3.6              pillar_1.4.7            
 [91] lifecycle_1.0.0          bitops_1.0-6             httpuv_1.5.5            
 [94] rtracklayer_1.50.0       R6_2.5.0                 latticeExtra_0.6-29     
 [97] promises_1.2.0.1         gridExtra_2.3            dichromat_2.0-0         
[100] MASS_7.3-53.1            assertthat_0.2.1         openssl_1.4.3           
[103] rprojroot_2.0.2          pinfsc50_1.2.0           withr_2.4.1             
[106] GenomicAlignments_1.26.0 GenomeInfoDbData_1.2.4   mgcv_1.8-34             
[109] hms_1.0.0                grid_4.0.3               rpart_4.1-15            
[112] bamsignals_1.22.0        rmarkdown_2.6            git2r_0.28.0            
[115] biovizBase_1.38.0        lubridate_1.7.9.2        base64enc_0.1-3         </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
